lib/assets/javascripts/views/plans/edit_details.js

```diff
@@ -20,4 +21,31 @@ $(() => {
     showHideDataContact($(e.currentTarget));
   });
   showHideDataContact($('#show_data_contact'));
+
+  // Keep the modal window's guidance selections in line with selections on the main page
+  const syncGuidance = () => {
+    const choices = $('#priority-guidance-orgs, #other-guidance-orgs').find('input[type="checkbox"]:checked')
```

Comment:
Since other-guidance-orgs is a modal you could have simplified the logic, i.e. you could have added a function handler for both containers (e.g. priority-guidance-orgs, other-guidance-orgs). Therefore if would be not needed:
- filter to remove duplicates (L29)
- Finding checkbox would be limited to the other container selector only, either #priority-guidance-orgs or #other-guidance-orgs.

Also, I would be very annoyed to have automatically disabled all my guidances if I select more than 6. I would go for a more event-based approach, perhaps custom event trigger when the MAX_NUMBER_GUIDANCE_SECTIONS is reached...